graphql/resolve/mutation_rewriter.go

```diff
@@ -1587,6 +1587,8 @@ func rewriteObject(
 		}
 	}
 
+	action := "update"
```

Comment:
Make it a constant

Reply:
We are changing the value of action based on whether the node is being created or already exists, so we can't make it a constant.